java/code/webapp/WEB-INF/pages/common/fragments/systems/system-header.jspf

```diff
@@ -52,7 +52,7 @@
 
 <rhn:require acl="formula_value_equals(system-lock, minion_blackout, true);not has_ssh_push_contact_method()">
 <div class="alert alert-warning">
-<p>System is locked. No actions are not allowed.</p>
+<p>System is locked. No actions are allowed.</p>
```

Comment:
Strictly speaking this is still not true, right? Or at least I saw a comment from @Bischoff about some things allowed.

Reply:
Only test.* and grains.* are allowed, but those are used internally.

Reply:
And moreover, test and grains are not actions.